ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -741,7 +741,21 @@ def inner(self, y, name=None):
 
     @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
     def mean(self, axis=None, keepdim=False, name=None):
-        return paddle_frontend.mean(self, axis=axis, keepdim=keepdim)
+        return paddle_frontend.mean(self._ivy_array, axis=axis, keepdim=keepdim)
```

Comment:
Why are we changing this? We can pass self directly to the frontend function because it is a frontend array. Morever, `_ivy_array` is a priavte property and we shouldn't be accessing it even though it will work since the frontend functions are capable of handling ivy arrays well. The most optimal approach is to just `self`.